pkg/kubelet/cpumanager/policy_static.go

```diff
@@ -48,15 +48,13 @@ func (p *staticPolicy) Name() string {
 }
 
 func (p *staticPolicy) Start(s state.State) {
-	// Build the initial shared cpuset.
-	// NB: Iteration starts at index `1` here because CPU `0` is reserved
-	//     for infrastructure processes.
-	// TODO(CD): Improve this to align with kube/system reserved resources.
-	shared := cpuset.NewCPUSet()
-	for cpuid := 1; cpuid < p.topology.NumCPUs; cpuid++ {
-		shared.Add(cpuid)
+	fullCpuset := cpuset.NewCPUSet()
+	for cpuid := 0; cpuid < p.topology.NumCPUs; cpuid++ {
+		fullCpuset.Add(cpuid)
 	}
-	s.SetDefaultCPUSet(shared)
+	// Figure out which cores shall not be used in shared pool
+	reserved, _ := takeByTopology(p.topology, fullCpuset, p.topology.NumReservedCores)
```

Comment:
Could we leave a note here saying that this depends on the order in which cores are taken in that function (low-numbered cores first?) We have to make this explicit in the contract so operators can reliably configure their systems around the CPU manager.

Glad this could be reused by the way, 👍